docs_src/design/ucr/Performance-Test.md

```diff
@@ -0,0 +1,68 @@
+## Performance Test
+
+## Submitters
+- Cherry Wang (IOTech)
+
+## Changelog
+<!--
+List the changes to the document, incl. state, date, and PR URL.
+
+State is one of: pending, approved, amended, deprecated
+Date is an ISO 8601 (YYYY-MM-DD) string.
+PR is the pull request that submitted the change, including information such as the diff, contributors, and reviewers.
+
+E.g.:
+- [approved](URL of PR) (2022-04-01)
+- [amended](URL of PR) (2022-05-01)
+-->
+
+## Market Segments
+- All
+
+## Motivation
+The EdgeX Performance Test aims to evaluate the efficiency of the EdgeX services in handling a large volume of data transmission from the south side to the north side in a manufacturing context. This use case is motivated by the need to understand how EdgeX performs under heavy data loads and to identify potential areas for optimization.
```

Comment:
Add the following sentences:
By providing the Performance Test Harness, users can execute the Performance Tests with theirown hardware and devices environment to evaluate the capability of the system.